lib/cog/adapters/hipchat/api.ex

```diff
@@ -14,7 +14,7 @@ defmodule Cog.Adapters.HipChat.API do
   end
 
   def lookup_room("@" <> handle) do
-    {:ok, user} = lookup_user(name: "@" <> handle)
+    {:ok, user} = lookup_user(handle: handle)
```

Comment:
Also, FWIW, "these functions use pattern matching but have totally different implementations and could potentially be given unique names" seems like the wrong way to go. Sharing a name should imply a common use / intent, independent of implementation. If a thing is properly named, I should be able to change the implementation and everything else still "makes sense".